Add documentation for wiffi integration

mampfes opened the pull request:
**Description:**


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30784

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

mampfes committed:
Add documentation for wiffi integration

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+---
+title: Wiffi
+logo: wiffi.png
+description: Support for Wiffi devices from stall.biz, e.g. Weatherman, Rainyman, ...
```
```suggestion
description: Support for WIFFI devices from stall.biz, e.g., Weatherman, Rainyman, ...
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+  - '@mampfes'
+---
+
+This integration allows you to connect your [Wiffi](https://stall.biz) devices directly to Home Assistant.
```
```suggestion
This integration allows you to connect your [STALL WIFFI](https://stall.biz) devices directly to Home Assistant.
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+
+Menu: *Configuration* > *Integrations*
+
+Press on **wiffi** and configure the integration:
```
```suggestion
Press on **WIFFI** and configure the integration:
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+
+Press on **wiffi** and configure the integration:
+
+* Enter the server port for json telegrams sent by the wiffi device.
```
```suggestion
* Enter the server port for JSON telegrams sent by the WIFFI device.
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+
+* Enter the server port for json telegrams sent by the wiffi device.
+
+## Configure wiffi device
```
```suggestion
## Configure the WIFFI device
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+
+## Configure wiffi device
+
+1. Set "CCU-IP Adresse myCCUIP" to IP address of Home Assistant.
```
```suggestion
1. Set "CCU-IP Adresse myCCUIP" to the IP address of Home Assistant.
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+## Configure wiffi device
+
+1. Set "CCU-IP Adresse myCCUIP" to IP address of Home Assistant.
+2. Set port for json telegrams to configured server port using parameter "send_json".
```
```suggestion
2. Set port for JSON telegrams to configured server port using parameter "send_json".
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+1. Set "CCU-IP Adresse myCCUIP" to IP address of Home Assistant.
+2. Set port for json telegrams to configured server port using parameter "send_json".
+
+Home Assistant will open a TCP server socket on the configured port and listens for incoming telegrams from wiffi devices. Entities from new devices will be automatically added.
```
```suggestion
Home Assistant will open a TCP server socket on the configured port and listens for incoming telegrams from WIFFI devices. Entities from new devices will be automatically added.
```

frenck commented on source/_integrations/wiffi.markdown:
```diff
@@ -0,0 +1,57 @@
+---
+title: Wiffi
```
```suggestion
title: STALL WIFFI
```

mampfes replied:
Done

frenck reviewed (changes requested):
Hi there @mampfes! 

I've reviewed your PR and left some comments, could you please take a look?

mampfes committed:
Use WIFFI / STALL WIFFI instead of wiffi to be consistent with stall.biz

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

mampfes committed:
Update HA release

mampfes committed:
update after review

remove configuration via yaml
copy text template for configuration via frontend
